all-builds.nix

```diff
@@ -55,7 +54,7 @@ let
   });
 
   perPlatform = lib.genAttrs cacheBuildSystems (system: let
-    reflex-platform = import ./dep/mars { inherit system __useNewerCompiler; };
```

Comment:
Gives me eval errors

Reply:
If you could post those errors that'd be great!

Reply:
I mean, the flag is gone on the haskell.nix branch. 
Compare
https://github.com/reflex-frp/reflex-platform/blob/2e0d31bc1b565eed02a3848cde0e0dbc4f33a2d4/default.nix#L12
with
https://github.com/reflex-frp/reflex-platform/blob/4fccfb196ac724695a5006a456b2125eeff331e5/default.nix#L2